applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -581,6 +589,13 @@ For example:
 
       west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
 
+
+To use nrf54H20 DK with the nrf7002ek shield attached (2.4 GHz or 5 GHz), follow the Developing with nRF7002 EB user guide to connect all required pins and then use the following command to build the sample:
```

Comment:
We need some editorial change and adding a hyperlink to the guide instead of plain text.
```suggestion
To use nrf54H20 DK with the ``nrf7002ek`` shield attached (2.4 GHz or 5 GHz), follow the :ref:`ug_nrf7002eb_nrf54h20dk_gs` user guide to connect all required pins and then use the following command to build the sample:
```